spec/src/main/asciidoc/fallback.asciidoc

```diff
@@ -21,7 +21,9 @@
 
 [[fallback]]
 == Fallback
-Fallbacks are invoked once a Retry or CircuitBreaker has failed enough times.
+Fallback provides an graceful alternative execution in case of failure.
+
+Fallbacks are invoked when used alone, once a Retry or CircuitBreaker has failed enough times.
```

Comment:
This line should be reworded
Fallback method is invoked when the method with `@Fallback` annotation completed exceptionally. `Fallback` annotation can be used on its own or together with other Fault Tolerance annotations.